### DISCUSSION TITLE
Adding SVG to Page Title

### DISCUSSION THREAD

**Discussion by fr34k4s01d** [requarks/wiki#3162] [Help / Questions] [answered]
Hi there,

i am curious to know, how to add a picture / icon ... SVG to the title of a page.
Like for example in https://docs.requarks.io/en/storage/git in the right corner.

Thanks in advance.

Best regards Christian

**Comment by moirasol** [comment]
I need to know the same! any idea?

**Comment by NGPixel** [comment, marked as answer]
You can see the source of any page by prefixing it with /s/

https://docs.requarks.io/s/en/storage/git

For the top-right corner icon, look at the very last line. It's simply a standard image with the `.align-abstopright` class.
